docs/source/docs/advanced-installation/sw_install/romi.md

```diff
@@ -0,0 +1,23 @@
+# Romi Installation
+
+The [Romi](https://docs.wpilib.org/en/latest/docs/romi-robot/index.html) is a small robot that can be controlled with the WPILib software. The main controller is a Raspberry Pi that must be imaged with [WPILibPi](https://docs.wpilib.org/en/latest/docs/romi-robot/imaging-romi.html) .
```

Comment:
do we even want to support this anymore? when was the last time this code was tested?

Reply:
I believe there is an issue open for updating it.

Reply:
That’s mine. Someone just confirmed that they were able to install on a ROMI with the new installer. I’ll update this section in another PR.